docs/available_checks.md

```diff
@@ -56,6 +56,7 @@ This document is a list of tables with a description and link to documentation f
 | [InvalidAccessTagCheck](checks/invalidAccessTagCheck.md) | The purpose of this check is to identify invalid access tags. |
 | [InvalidLanesTagCheck](docs/checks/invalidLanesTagCheck.md) | The purpose of this check is to identify highways in OSM with an invalid lanes tag value. |
 | [MixedCaseNameCheck](checks/mixedCaseNameCheck.md) | The purpose of this check is to identify names that contain invalid mixed cases so that they can be edited to be the standard format. |
+| [RoadNameGapCheck](checks/RoadNameGapCheck.md) | The purpose of this check is to identify edge connected between two edges whose name tag is same. Flag the edge if the edge has a name tag different to name tag of edges connected to it or if there is no name tag itself.
```

Comment:
typo: The purpose of this check is to identify edge connected between two edges whose name tag is ~same~ different.

Reply:
Actually, it is not a typo. I need to find an edge which is connected between two edges whose name tag is same. IF the intermediate edge has no name then flag it. If the edge has a name but is different to the matching in and out edge then check if the edge has any other connected edge which has same name. If no then create a flag.